lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -383,7 +399,7 @@ export class PublicClientApplication implements IPublicClientApplication {
                     ...silentRequest,
                     redirectUri: request.redirectUri,
                     prompt: PromptValue.NONE
-                });
+                }, InteractionType.SILENT);
```

Comment:
Would we ever care about differentiating between SilentFlow and SilentHandler?

Reply:
They are both calling the same helper which creates SilentHandler, so I don't think it is necessary.